inference-engine/src/transformations/include/transformations/convert_extract_image_patches_to_reorg_yolo.hpp

```diff
@@ -0,0 +1,30 @@
+// Copyright (C) 2018-2020 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#pragma once
+
+#include <vector>
+#include <memory>
+
+#include <ie_api.h>
```

Comment:
Please, merge changes from master. There were some changes related to transformations implementation and the current version may not work.

Reply:
Once again. Use
`#include <transformations_visibility.hpp>`
instead of 
`#include <ie_api.h>`

Your code even is not compilable.

Reply:
Just take a look at for example, inference-engine/src/transformations/include/transformations/convert_batch_to_space.hpp